contentgrid-spring-data-rest/src/testFixtures/java/com/contentgrid/spring/test/fixture/invoicing/model/Order.java

```diff
@@ -34,7 +35,7 @@ public class Order {
     @JsonProperty(access = Access.READ_ONLY)
     private UUID id;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
I don't think cg-apps have `FetchType.LAZY` for `@ManyToOne`  relations right now ? why are we changing this here ?